graph-builder/src/plugins/release_scrape_dockerv2/registry.rs

```diff
@@ -193,10 +181,10 @@ pub async fn fetch_releases(
     repo: &str,
     username: Option<&str>,
     password: Option<&str>,
-    cache: &mut crate::registry::cache::Cache,
+    cache: &mut cache::Cache,
```

Comment:
Could `cache::CacheAsync` be used here from the start?

Reply:
Yes. It would require to be cloned for every plugin invocation, which might be even better than using interior mutability as we do now. Any strong preferences?

Reply:
I don't think there is any benefit in having (sync) `Cache` and `CacheAsync`, so there should be only one public - async one. Plugins should probably use their own cache copies

Reply:
There's only a single public cache type which can be instantiated with `cache::new`. I think this design is fine for the use-case of passing in a cache from the outside as well, which needs to be possible for implementing cache persistence in a separate module.

Reply:
@vrutkovs resolved?